aptos-move/e2e-move-tests/tests/rotate_auth_key.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_crypto::ed25519::Ed25519PrivateKey;
+use aptos_crypto::{PrivateKey, SigningKey, Uniform};
+use aptos_types::{
+    account_address::AccountAddress, account_config::CORE_CODE_ADDRESS,
+    state_store::state_key::StateKey, state_store::table::TableHandle,
+    transaction::authenticator::AuthenticationKey,
+};
+
+use cached_framework_packages::aptos_stdlib;
+use e2e_move_tests::{assert_success, enable_golden, MoveHarness};
+use move_deps::move_core_types::parser::parse_struct_tag;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+struct OriginatingAddress {
+    handle: u128,
+}
+
+#[derive(Serialize, Deserialize)]
+struct Proof {
+    account_address: AccountAddress,
+    module_name: String,
+    struct_name: String,
+    originator: AccountAddress,
```

Comment:
What's the difference between `account_address` and `originator`?

Reply:
`account_address` refers to the account that owns the struct `RotationProof` - which is our `CORE_CODE_ADDRESS` / `aptos_framework` address. 

the originator is the user's first address.

Reply:
Cool! Please add this to the comments!